client/components/header-nav/README.md

```diff
@@ -0,0 +1,4 @@
+Header Nav
```

Comment:
I feel like this should be named differently. To me it means "Masterbar Secondary Navigation", but that's a bit long :)